### ISSUE TITLE
blob storage cannot open container

### ISSUE THREAD

**Issue by chen112p** [microsoft/AzureStorageExplorer#5450]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.22

### Architecture

i86

### Platform

Windows

### Bug Description

Unable to retrieve child resources.

Details:
{
  "name": "FetchError",
  "message": "request to https://labsjrcc.blob.core.windows.net/?comp=list&include=metadata failed, reason: connect ETIMEDOUT 20.150.20.132:443",
  "stack": "FetchError: request to https://labsjrcc.blob.core.windows.net/?comp=list&include=metadata failed, reason: connect ETIMEDOUT 20.150.20.132:443\n    at ClientRequest.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\node-fetch\\lib\\index.js:1491:11)\n    at ClientRequest.emit (events.js:315:20)\n    at TLSSocket.socketErrorListener (_http_client.js:469:9)\n    at TLSSocket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
  "type": "system",
  "errno": "ETIMEDOUT",
  "code": "ETIMEDOUT"
}

### Steps to Reproduce

1. Launch storage explorer
2. expand storage account
3. expand the blob storage of "cool-storage", the having the reported error
4. Everything else is ok, and other people in my team who has the subscription doesnt have this issue

### Actual Experience

Cannot access data from a particular blob storage from the storage explorer. I can see the containers from the azure portal, can create SAS tokens, but cannot see whats inside

**Comment by MRayermannMSFT** [comment]
@chen112p can you give system proxy a try? Read more about it here: https://docs.microsoft.com/en-us/azure/storage/common/storage-explorer-network#use-system-proxy-preview

**Comment by chen112p** [comment]
Thanks for responding! The issue resolved by itself. But I experienced the same issue once again and once again it resolved by itself within a shorter period of time. First time it was resolved in a bit more than 2 days. This time it was resolved in 10hours. 
Now I can confirm I saw this issue once I tried to remove a large amount of data from the Storage Explorer (first time >300Gb, second time >100Gb). Then the Storage Explorer shows a percentage of deletion progress in the 'activities' window, and it got stuck at certain percentage. When I tried to open another container during this deletion process, I saw an error bar saying lost connection or something, and once I tried to close the Storage Explorer, it tells me something will go wrong if I went ahead closing it off. And indeed it went wrong. 

Sorry about the lengthy explanation. Now I know probably a better option to remove large amount of data from blob storage would be using 'azcopy remove'. Thanks for your patients
